ASPectLibrary/ApplicationUser.cs

```diff
@@ -11,5 +11,6 @@ public class ApplicationUser : IdentityUser {
 
         public ICollection<Enrollment> Enrollments { get; set; }
         public IList<Membership> Memberships { get; set; }
+        public IList<Offering> Offerings { get; set; }
```

Comment:
Offerings array is still lazy loaded, can you eager load this?

Reply:
![image](https://user-images.githubusercontent.com/40650969/114331283-8b072400-9af8-11eb-93ed-6c991a5c54ea.png)

Reply:
Offerings shouldn't be null anymore if an instructor does have an Offering.